serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -181,9 +181,33 @@ export class _GLTFMaterialExporter {
             }
         }
 
+        // Do a naieve check for materials that contain extensions that have texture-related members.

```

Comment:
I'm not a fan of this implementation, but it seems effective. I also considered adding a mapping if which IMaterials include textures that is created as the IMaterials are created, I can change if there are any serious considerations of the efficiency of iterating over all members in our exported materials.

This could use some unit tests for the various extensions that include texture information.

Reply:
Refactored the implementation to be more object oriented.